lib/scnnr/polling_utils.rb

```diff
@@ -0,0 +1,37 @@
+# frozen_string_literal: true
+
+module Scnnr
+  class PollingUtils
```

Comment:
Sorry, I don't understand why we need to refactor the current class in this static way.
If you have any opinion, could you explain that?